### PR TITLE
[10.0][FIX] l10n_it_corrispettivi: Find fiscal positions without company

### PR DESCRIPTION
**Pull request by SimoRubi**
Fix https://github.com/OCA/l10n-italy/issues/1156

--
Confermo di aver firmato il CLA https://odoo-community.org/page/cla e di aver letto le linee guida su https://odoo-community.org/page/contributing

### CODE REVIEW

File: l10n_it_corrispettivi/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
 {
     'name': 'Italian Localization - Corrispettivi',
-    'version': '10.0.1.2.2',
+    'version': '10.0.1.3.0',
```
Comment (tafaRU) [inline l10n_it_corrispettivi/__manifest__.py]: come mai non `10.0.1.2.3`?
Mi pare ricada in:
`z increments when bugfixes were made. Usually a server restart is needed for the fixes to be made available.`(cit. [OCA version-numbers](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers))

Comment (eLBati) [reply l10n_it_corrispettivi/__manifest__.py]: Penso anch'io

Comment (SimoRubi) [reply l10n_it_corrispettivi/__manifest__.py]: @tafaRU grazie della review! In effetti ero indeciso, ho messo `10.0.1.2.3`

**Comment by eLBati** [comment]
@SimoRubi fai un unico commit plis

**Comment by SimoRubi** [comment]
@eLBati ok, fatto

**Pull request merged**